jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -28,6 +35,15 @@ class UserProfile extends React.Component {
   }
 
   /**
+    Set whether the profile has entities.
```

Comment:
Please consider the change above, and also describe better how and where this is set.